world.js

```diff
@@ -292,6 +292,7 @@ const _getBindSceneForRenderPriority = renderPriority => {
 const _bindHitTracker = app => {
   const hitTracker = hpManager.makeHitTracker();
   hitTracker.bind(app);
+  app.dispatchEvent({type: 'bindHitTracker'});
```

Comment:
Can we rename this to `hittrackeradded` or `hittrackerattached`? Something that makes it clear the event already happened. Not "please perform a bind".

Reply:
Renamed https://github.com/webaverse/app/pull/2933/commits/dd4777c796d94f0c07325eedd0fd0dbb3a56979a